src/main/docker/kafdrop.sh

```diff
@@ -65,4 +65,4 @@ ARGS="--add-opens=java.base/sun.nio.ch=ALL-UNNAMED -Xss256K \
      $HEAP_ARGS \
      $JVM_OPTS"
 
-exec java $ARGS -jar /kafdrop*/kafdrop*jar ${CMD_ARGS}
+exec java $ARGS -jar /opt/kafdrop/kafdrop*.jar ${CMD_ARGS}
```

Comment:
Due to the above mentioned changes, this path needs to be changed:
```suggestion
exec java $ARGS -jar /opt/kafdrop/kafdrop*/kafdrop*jar ${CMD_ARGS}
```